home.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <script src="script.js" defer></script>
+    <title>Polanco Studio</title>
+  </head>
+  <body>
+    <div id="myNav" class="overlay">
+      <a href="javascript:void(0)" class="closebtn">&times;</a>
+      <div class="overlay-content">
+        <a href="#programa" id="mybtn">Program</a>
+        <a href="#about">About</a>
+        <a href="#gallery">Gallery</a>
+      </div>
+    </div>
+    <div class="topbar">
+      <ul>
+        <li>
+          <a href="">
+            <img class="topbar-icon" src="./img/fb.svg" alt="Fb logo" />
+          </a>
+        </li>
+        <li>
+          <a href="">
+            <img
+              class="topbar-icon"
+              src="./img/Twitter.svg"
+              alt="Twitter logo"
+            />
+          </a>
+        </li>
+        <li>Logout</li>
+        <li>Mypage</li>
+        <li>Korean</li>
+      </ul>
+    </div>
+    <nav>
+      <div class="image-nav">
+        <img src="./img/logo-footer.png" alt="Image Logo" />
+      </div>
+      <ul>
+        <li>
+          <a href="about.html">About</a>
+        </li>
+        <li>
+          <a href="#programa">Program</a>
+        </li>
+        <li>
+          <a href="#gallery">Gallery</a>
+        </li>
+        <li>
+          <a href="#">News</a>
+        </li>
+        <li>
+          <a href="#">Join</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="backimg">
+      <header>
+        <a href="#" id="Hamb" class="xNav">☰</a>
+      </header>
+      <main>
+        <div class="welcome">
+          <h2>"Hello! sharing world"</h2>
+          <h1>
+            ESTUDIO POLANCO GLOBAL SUMMIT <br />
+            2021
+          </h1>
+        </div>
+        <div class="paragraph">
```

Comment:
- [ ] As required, please stick to these colors of the given design.
* Basic color swatch: #272A31, #EC5242, #D3D3D3

Reply:
fixed 👍